rest-api/src/test/java/gov/cms/qpp/conversion/api/controllers/v1/ExceptionHandlerControllerV1Test.java

```diff
@@ -224,6 +231,30 @@ void testHandleAmazonExceptionResponseBodyDoesInterception() throws Exception {
 		//mvc.perform(RequestBuilder("/cpc/unprocessed-files"));
 	}
 
+	@Test
+	void testHandleInvalidPurposeExceptionExceptionResponseBody() {
+		InvalidPurposeException exception = new InvalidPurposeException("some message");
+
+		ResponseEntity<String> response = objectUnderTest.handleInvalidPurposeException(exception);
+
+		Truth.assertThat(response.getBody()).contains("some message");
+	}
+
+	@Test
+	void testHandleInvalidPurposeExceptionResponseBodyDoesInterception() throws Exception {
+		QrdaControllerV1 mock = Mockito.mock(QrdaControllerV1.class);
+		MockMvc mvc = MockMvcBuilders.standaloneSetup(mock)
+				.setControllerAdvice(new ExceptionHandlerControllerV1(auditService))
+				.build();
+		Mockito.when(mock.uploadQrdaFile(ArgumentMatchers.any(), ArgumentMatchers.anyString())).thenCallRealMethod();
+
+		RequestBuilder builder = MockMvcRequestBuilders.fileUpload("/")
+			.file("file", ArrayUtils.EMPTY_BYTE_ARRAY)
+			.header("Purpose", "this is an invalid purpose because it's too long" + UUID.randomUUID());
+		MvcResult result = mvc.perform(builder).andReturn();
+		Truth.assertThat(result.getResponse().getContentAsString()).isEqualTo("Given Purpose (header) is too large");
```

Comment:
Looks like CircleCI failed on this test here.  This'll need to be updated since you just changed the error response to mention the maximum allowed size.